tests/test_yaml_parser.py

```diff
@@ -1,13 +1,21 @@
+from typing import Tuple
 import pytest
 from pathlib import Path
 
 from hesiod.cfgparse import YAMLConfigParser
 
 
-def test_load_cfg_file() -> None:
+@pytest.fixture
+def paths() -> Tuple[Path, Path]:
     cwd = Path(".").absolute()
-    run_cfg_file = cwd / "tests/cfg/dataset/cifar/cifar100.yaml"
     base_cfg_dir = cwd / "tests/cfg"
+    runs_cfg_dir = cwd / "tests/runs"
+    return base_cfg_dir, runs_cfg_dir
+
+
+def test_load_cfg_file(paths: Tuple[Path, Path]) -> None:
+    base_cfg_dir = paths[0]
```

Comment:
Secondo me i test si possono ancora migliorare. Vedo molti casi in cui `base_cfg_dir` non mi sembra usato. Inoltre la fixture `paths` e' molto. generica, sarebbe bello che un test usasse una fixture ben capibile. Per esempio, un test che deve verificare che il parser fallisce con uno YAML malformato potrebbe usare una fixture `malformed_yaml`